package.json

```diff
@@ -5,16 +5,16 @@
   "main": "index.js",
   "scripts": {
     "start": "BASE_HREF=/ node server",
+    "test": "npm run unit && npm run lint && npm run build",
     "lint": "eslint app react test ./*.js",
-    "test": "BABEL_ENV=test mocha 'react/**/*.test.js' --compilers js:babel-register --require test/setup.js && npm run lint && npm run build",
-    "build": "rm -rf dist/ && mkdir dist && NODE_ENV=production BASE_HREF=/ webpack --config webpack.static.render.config.js && rm dist/render.js && NODE_ENV=production webpack --config webpack.static.client.config.js",
-    "build-gh-pages": "rm -rf dist/ && mkdir dist && NODE_ENV=production BASE_HREF=/seek-style-guide/ webpack --config webpack.static.render.config.js && rm dist/render.js && NODE_ENV=production webpack --config webpack.static.client.config.js",
-    "preversion": "npm run build-gh-pages",
-    "postversion": "npm publish && git push && git push --tags && npm run deploy",
-    "deploy": "gh-pages -d dist && printf '\nURL: https://seek-oss.github.io/seek-style-guide\n\n'",
-    "deploy-surge": "surge -p ./dist",
+    "unit": "BABEL_ENV=test mocha 'react/**/*.test.js' --compilers js:babel-register --require test/setup.js",
```

Comment:
Very mildly prefer the Company Reviews (and others?) convention of calling this `test-unit`.  Probably less important if you don't have too many other types of tests.